src/main/kotlin/org/rust/lang/core/resolve/ref/RsPathReference.kt

```diff
@@ -12,8 +12,6 @@ import org.rust.lang.core.types.BoundElement
 interface RsPathReference : RsReference {
     override fun getElement(): RsPath
 
-    fun resolveIfVisible(): RsElement? = multiResolveIfVisible().singleOrNull()
```

Comment:
@dima74 did you add this method intentionally as a part of API?

Reply:
Yes, so I think better to suppres it as unused for now